### PR TITLE
Change parameter name to enable Hypercore TAM

### PR DESCRIPTION
**Pull request by mkindahl**
Changing from using the `compress_using` parameter with a table access method name to use the boolean parameter `hypercore_use_access_method` instead to avoid having to provide a name when using the table access method for compression.

### CODE REVIEW

**Review by kpan2034 — APPROVED** [review]

**Review by erimatnor — APPROVED** [review]
Just one question about name of configuration in policy json. Should it be the same as the parameter name in compress_chunk?

**Comment by mkindahl** [comment]
> Just one question about name of configuration in policy json. Should it be the same as the parameter name in compress_chunk?

Yes, I think so. All user-facing parameters should be coordinated. Will fix.

File: tsl/src/bgw_policy/policies_v2.h [hunk 1]
```diff
@@ -25,7 +26,7 @@
 #define POL_COMPRESSION_CONF_KEY_COMPRESS_AFTER "compress_after"
 #define POL_COMPRESSION_CONF_KEY_MAXCHUNKS_TO_COMPRESS "maxchunks_to_compress"
 #define POL_COMPRESSION_CONF_KEY_COMPRESS_CREATED_BEFORE "compress_created_before"
-#define POL_COMPRESSION_CONF_KEY_COMPRESS_USING "compress_using"
+#define POL_COMPRESSION_CONF_KEY_USE_ACCESS_METHOD "use_access_method"
```
[the remark this answers is not in the document]
Comment (mkindahl) [reply tsl/src/bgw_policy/policies_v2.h]: Added the prefix.

**Comment by mkindahl** [comment]
> > Just one question about name of configuration in policy json. Should it be the same as the parameter name in compress_chunk?
> 
> Yes, I think so. All user-facing parameters should be coordinated. Will fix.

Done.

**Pull request merged**